### PR TITLE
github(PULL_REQUEST_TEMPLATE): Add pull request template to Centreon Broker

### PR DESCRIPTION
**Pull request by adr-mo**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,44 +1,98 @@
...
+***Versions***
+
+For the RPM based systems
+
+-- Copy/Paste the result of the following command --
+```
+$ rpm -qa | grep centreon
+```
+
+For the deb based systems
```
Comment (ganoze) [inline .github/ISSUE_TEMPLATE.md]: For deb based systems, there are no official packages. You might need to explain that we need the installation method of the components (sources, third party package, ...) and the version associated with each component.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,44 +1,98 @@
...
+
+-- Copy/Paste the result of the following command --
+```
+$ dpkg -l | grep centreon
+```
+***Operating System***
+
+*CentOS, Debian ...*
+
+***Browser used***
```
Comment (ganoze) [inline .github/ISSUE_TEMPLATE.md]: Irrelevant for Centreon Broker ?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,44 +1,98 @@
...
-1.
-2.
-3.
+### Logs
 
+**PHP error logs**
 
-**Describe the results you received:**
+```
+tail -f /var/opt/rh/rh-php71/log/php-fpm/centreon-error.log
```
Comment (ganoze) [inline .github/ISSUE_TEMPLATE.md]: _rh-php72_ is a more up-to-date target.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (adr-mo) [reply .github/ISSUE_TEMPLATE.md]: Done :)

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (adr-mo) [reply .github/ISSUE_TEMPLATE.md]: done

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (adr-mo) [reply .github/ISSUE_TEMPLATE.md]: done

**Pull request merged**